djangocms_blog/models.py

```diff
@@ -232,7 +232,7 @@ def get_title(self):
         return title.strip()
 
     def get_keywords(self):
-        return self.safe_translation_getter('meta_keywords').strip().split(',')
+        return self.safe_translation_getter('meta_keywords', default='').strip().split(',')
```

Comment:
Argh! So that's the problem that causes the issue!
So, the stronger check have even more sense.